### PR TITLE
TES2_14 Moving Test to JDBC

### PR DESCRIPTION
**Pull request by pratikzode**
Migrating the extension installcheck tests into the JDBC framework.
bb installcheck-extensions --extension=babelfishpg_tsql

File Change:
There was no major change in the files. Have removed the test cases which was not working and not currently supported in Babelfish. 
babel_219.sql
babel_collection.sql
babel_datatype.sql
babel_ddl.sql
babel_delete1.sql
babel_emoji.sql
babel_function.sql
babel_like.sql
babel_set_command.sql
babel_table_type.sql
babel_transaction.sql
babel_typecode.sql
babel_uniqueidentifier.sql

Have removed some of the test like:
```
create domain public.varchar as pg_catalog.varchar(2) check (char_length(value) < 1);
GO
ERROR (Code: 33557097)

ERROR (Message: syntax error near 'domain' at line 4 and character position 7
```
```
alter table testing1 alter column col nvarchar(60) collate Arabic_CS_AS;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: 'COLLATE in ALTER TABLE ALTER COLUMN' is not currently supported in Babelfish)~~
```
```
reset babelfishpg_tsql.sql_dialect;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: syntax error near '.' at line 1 and character position 22)~~
```

```
set babelfishpg_tsql.sql_dialect = 'tsql';
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: syntax error near 'sql_dialect' at line 1 and character position 21)~~
```
```
select CAST(2 AS numeric) > 1;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: syntax error near '>' at line 2 and character position 26)~~
```
```

CREATE EXTENSION IF NOT EXISTS "babelfishpg_tsql" CASCADE;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: syntax error near 'EXTENSION' at line 4 and character position 7)~~
```
```

select PARSE('2017-08-25' AS date);
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: 'PARSE' is not currently supported in Babelfish)~~
```

Have changed `explain (costs false)` to `babelfish_showplan_all`because babelfish does not support explain
```
explain (costs false) select c1 from testing4 where c1 LIKE '_one_';
```

Task: TES2_14-jira-babel
Signed-off-by: Pratik Zode [pzode@amazon.com](mailto:pzode@amazon.com)

### CODE REVIEW

**Review by forestkeeper — DISMISSED** [review]
LGTM

File: test/JDBC/expected/babel_219.out [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+GO
+~~START~~
+varchar
+
+~~END~~
+
+create table t1(col varchar);
+insert into t1 (col) select 'a'; -- fail
+insert into t1 (col) select ''; -- pass
+select * from t1;
```
Comment (macwoody) [inline test/JDBC/expected/babel_219.out]: order by

File: test/JDBC/expected/babel_219.out [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+~~END~~
+
+
+-- verify behavior of public.varchar is unchanged in tsql dialect
+create table t2(col varchar);
+GO
+insert into t1 (col) select 'a'; -- fail
+insert into t1 (col) select ''; -- pass
+select * from t1;
```
Comment (macwoody) [inline test/JDBC/expected/babel_219.out]: order by

File: test/JDBC/expected/babel_219.out [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+varchar
+a
+
+~~END~~
+
+
+-- verify behavior of public.varchar is unchanged in tsql dialect
+create table t2(col varchar);
+GO
+insert into t1 (col) select 'a'; -- fail
```
Comment (macwoody) [inline test/JDBC/expected/babel_219.out]: why create a t2 and insert into t1 ?

File: test/JDBC/expected/babel_delete1.out [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+        (34, 'fname2', 'lname2', 'paris'),
+        (35, 'fname3', 'lname3', 'brussels'),
+        (90, 'fname4', 'lname4', 'new york'),
+        (26, 'fname5', 'lname5', 'los angeles'),
+        (74, 'fname6', 'lname6', 'tokyo'),
+        (44, 'fname7', 'lname7', 'oslo'),
+        (19, 'fname8', 'lname8', 'hong kong'),
+        (61, 'fname9', 'lname9', 'shanghai'),
+        (29, 'fname10', 'lname10', 'mumbai');
+SELECT * FROM delete_test_tbl;
```
Comment (macwoody) [inline test/JDBC/expected/babel_delete1.out]: order by

File: test/JDBC/expected/babel_delete1.out [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+74#!#fname6    #!#lname6    #!#tokyo               
+44#!#fname7    #!#lname7    #!#oslo                
+19#!#fname8    #!#lname8    #!#hong kong           
+61#!#fname9    #!#lname9    #!#shanghai            
+29#!#fname10   #!#lname10   #!#mumbai              
+~~END~~
+
+
+-- Prove that a user may delete rows from a table without using the FROM clause
+SELECT * FROM delete_test_tbl;
```
Comment (macwoody) [inline test/JDBC/expected/babel_delete1.out]: order by

File: test/JDBC/expected/babel_delete1.out [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+29#!#fname10   #!#lname10   #!#mumbai              
+~~END~~
+
+
+-- Test that that WHERE clause can be used without FROM
+DELETE delete_test_tbl WHERE city='hong kong';
+GO
+~~ROW COUNT: 1~~
+
+SELECT * FROM delete_test_tbl;
```
Comment (macwoody) [inline test/JDBC/expected/babel_delete1.out]: order by

File: test/JDBC/expected/babel_delete1.out [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+61#!#fname9    #!#lname9    #!#shanghai            
+29#!#fname10   #!#lname10   #!#mumbai              
+~~END~~
+
+
+DELETE delete_test_tbl WHERE age > 50;
+GO
+~~ROW COUNT: 3~~
+
+SELECT * FROM delete_test_tbl;
```
Comment (macwoody) [inline test/JDBC/expected/babel_delete1.out]: order by

File: test/JDBC/expected/babel_delete1.out [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+44#!#fname7    #!#lname7    #!#oslo                
+29#!#fname10   #!#lname10   #!#mumbai              
+~~END~~
+
+
+DELETE delete_test_tbl WHERE fname IN ('fname1', 'fname2');
+GO
+~~ROW COUNT: 2~~
+
+SELECT * FROM delete_test_tbl;
```
Comment (macwoody) [inline test/JDBC/expected/babel_delete1.out]: order by

File: test/JDBC/expected/babel_select_distinct_top.out [hunk 9]
```diff
@@ -0,0 +1,59 @@
+create table babel_select_distinct_top (a int);
+GO
+insert into babel_select_distinct_top values (3), (1), (2), (2), (1);
+GO
+~~ROW COUNT: 5~~
+
+select * from babel_select_distinct_top;
```
Comment (macwoody) [inline test/JDBC/expected/babel_select_distinct_top.out]: order by

File: test/JDBC/expected/babel_typecode.out [hunk 10]
```diff
@@ -0,0 +1,40 @@
+-- test typecode list sys table
+SELECT pg_namespace, pg_typname, tsql_typname, type_family_priority, priority, sql_variant_hdr_size FROM sys.babelfish_typecode_list();
```
Comment (macwoody) [inline test/JDBC/expected/babel_typecode.out]: order by

**Review by macwoody — APPROVED** [review]

**Pull request merged**